lib/convection/control/stack.rb

```diff
@@ -248,6 +275,22 @@ def validate
         puts "\nTemplate validated successfully"
       end
 
+      def after_create_task(task)
+        @tasks[:after_create].unshift(task)
```

Comment:
@simonirwin-r7 @abegley-r7 @erran-r7 resolved in 84bc2cf04381c5042d0716634b677a1b9a2f6f62.